hawkbit-repository/hawkbit-repository-core/src/main/java/org/eclipse/hawkbit/repository/RolloutHelper.java

```diff
@@ -253,4 +256,12 @@ public static void checkIfRolloutCanStarted(final Rollout rollout, final Rollout
                     + rollout.getStatus().name().toLowerCase());
         }
     }
+
+    public static boolean isRolloutRetried(final String targetFilter) {
+        return targetFilter.contains("failedRollout");
```

Comment:
is this case sensitive? 
aren't lower cased attributes are supported at the moment? e.g. "createdat"
if search attributes are case insensitive this check shall be also such.
If not - better use lowercase as in other cases